example/src/App.tsx

```diff
@@ -41,6 +41,37 @@ export type RootStackParamsList = {
 const RootStack = createNativeStackNavigator();
 
 export default function App() {
+  var stackParams = {
+    data: [
+      {
+        title: 'Basic playback',
+        routeName: 'BasicPlayback',
+      },
+      {
+        title: 'Basic Drm playback',
+        routeName: 'BasicDrmPlayback',
+      },
+      {
+        title: 'Subtitle and captions',
+        routeName: 'SubtitlePlayback',
+      },
+      {
+        title: 'Basic Picture in Picture',
+        routeName: 'BasicPictureInPicture',
+      },
+      {
+        title: 'Basic Ads',
+        routeName: 'BasicAds',
+      },
+    ],
+  };
+
+  if (!Platform.isTV) {
```

Comment:
Does this include Android TV?

Reply:
nice catch, fixed in 69e9e99bdd704ab6cadfa0dec4ddb6bcc3c599cf